_posts/en/2020-12-04-better-tech-through-micro-procurementpt2.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title:  "Better tech through micro-procurement: part two"
+ref: micro-procurement-p2
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team, and procurement experts at ESDC"
+date: "2020-12-04"
+---
+<!--markdownlint-disable MD033-->
+This is part two of a two-part series looking at micro-procurement for digital work.
+In this second post, we look at what can make implementing micro-procurement challenging, as well as a sneak peek at what we are planning at ESDC.
+
+***
+
+## What’s stopping the GC from doing micro-procurement of digital work
+
+The web of procurement rules is real.
+Procurement officers with extensive training help us navigate that web.
+They help ensure that things are done by the book and no-one loses their financial signing authority or worse, gets fired.
+Like all policy, though, procurement policy is open to interpretation and because of this, appetite for risk (by that procurement officer, procurement team, CFO or Deputy Head) can be a factor in implementing any procurement innovation.
+It can be seen as less risky to do things the way they have always been done.
+Related, a "fear of audit" was noted in the [Internal Red Tape Reduction report](https://internal-red-tape-reduction-report.github.io/chapter-3/#root-causes) as a driver of "the need to document every step of the process, which, in turn, drives the heavy paperwork, all in paper format."
+But as the Office of the Procurement Ombudsman pointed out in their most recent [annual report](http://opo-boa.gc.ca/rapports-reports/2019-2020/index-eng.html), simplifying and streamlining procurement in the GC "may require a higher degree of risk tolerance.”
+
+There is another challenge related to procurement policy.
+We know that flexibility exists within GC procurement policy to establish a simplified process for LDV procurements.
```

Comment:
Spell out LDV as its first time seen on the page.

Reply:
Good catch! Fixed. :)